src/base/io/utilities/xls2model.m

```diff
@@ -151,32 +185,31 @@
     %Default -1000
     lowerBoundList(isnan(lowerBoundList)) = -1000;
 else
-    lowerBoundList = -defaultbound*ones(length(rxnAbrList),1);
+    lowerBoundList = 1000*ones(length(rxnAbrList),1);

```

Comment:
Because this changed at some point and since I was basing a lot of my stuff on my FileIOUpdate branch, I simply overwrote the whole file, corrected now, along with an additional fix for IDs which are numbers.